### PR TITLE
fix: evm amount is now calculated as user input minus the fee

### PR DESCRIPTION
**Pull request by mpetrunic**
## Related Issue Or Context

Closes: #139 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in sygma-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Review by wainola — CHANGES_REQUESTED** [review]

**Review by wainola — DISMISSED** [review]

**Comment by mpetrunic** [comment]
Depends on: https://github.com/sygmaprotocol/sygma-sdk/pull/360

**Comment by mpetrunic** [comment]
This is blocked until, testnet fee handlers are deployed to use latest smart contracts

**Comment by mpetrunic** [comment]
depends on https://github.com/sygmaprotocol/sygma-sdk/pull/365

File: packages/widget/src/controllers/transfers/evm/build.ts [hunk 1]
```diff
@@ -28,14 +33,48 @@ export async function buildEvmFungibleTransactions(
...
   const transfer = await evmTransfer.createFungibleTransfer(
     address,
     this.destinationNetwork.chainId,
     this.destinationAddress,
     this.selectedResource.resourceId,
-    String(this.resourceAmount)
+    this.resourceAmount.toString()
   );
   const fee = await evmTransfer.getFee(transfer);
+  transfer.details;
```
Comment (Lykhoyda) [inline packages/widget/src/controllers/transfers/evm/build.ts:77]: it's probably not reqired

**Review by Lykhoyda — DISMISSED** [review]
Looks good!

**Review by wainola — APPROVED** [review]

**Review by MakMuftic — APPROVED** [review]

**Pull request merged**